mysql/Dockerfile

```diff
@@ -7,8 +7,10 @@ RUN mkdir /etc/service/mysql && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 COPY my.cnf /etc/mysql/my.cnf
+COPY mysql.mk /usr/share/tugboat/mysql.mk
```

Comment:
Let's put this in the baseimage with the rest of the main Makefile. The ability to call `make install-mysql-X` should be allowed from any of our images, but would probably be most likely to be called from the ubuntu image.

Reply:
Wouldn't someone only want to install or upgrade mysql on a mysql image though? What would be the use case for installing mysql on another service?

Reply:
If the primary MySQL service doesn't have the version you want, you could use the vanilla "ubuntu" image, and install the one you want.

Also, if for some strange reason your code expects MySQL to be at localhost, we should allow users to install it on the webhead, even if we think it's not a great idea.

Basically, there's no harm in allowing the flexibility of having this available on all services, so there's no good reason to impose restrictions of where we think it _should_ be used.